psycop/common/sequence_models/embedders/BEHRT_embedders.py

```diff
@@ -199,7 +199,17 @@ def get_mapped_diagnosis_codes(
         ) as fp:
             mapping = json.load(fp)
 
-        return [mapping[d] for d in diagnosis_codes if d in mapping]
+        # For each diagnosis code, attempt to map to caliber code
```

Comment:
A tiny nitpick. The method is not a 'getter' because it transforms the value. I'd rename to something like `map_icd_to_calibre` :-)